packages/early-hints/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@cloudflare/early-hints-plugins",
+  "version": "1.0.0",
+  "main": "index.js",
+ "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts",
+    "tsconfig.json"
```

Comment:
Copy in the `tsconfig.json` from [here](https://github.com/cloudflare/pages-plugins/blob/main/packages/google-chat/tsconfig.json).